README.md

```diff
@@ -91,13 +91,9 @@ FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
 2. [General Installing Instructions](General-Installing-Instructions.md)
 
 3. Installing Cacti on Linux
-
-    3.1.  [Installation Under CentOS 7 - LAMP Stack](Install-Under-CentOS_LAMP.md)
-    
-    3.2.  [Installation Under CentOS 7 - LEMP Stack](Install-Under-CentOS_LEMP.md)
-    
-    3.3. [Installation Under Ubuntu/Debian - LAMP Stack](Installing-Under-Ubuntu-Debian.md)
-
+    - 3.1.  [Installation Under CentOS 7 - LAMP Stack](Install-Under-CentOS_LAMP.md)
```

Comment:
Not sure, but I think there are too many spaces here.  The minus should line up with the Installing (a bit like in yaml).

Reply:
Also, not sure if there should be a blank line between each element.

Reply:
There shouldn't be a mixture of - and numbered lists.

Reply:
Now I am wondering what you were attempting to fix?